### PR TITLE
Add role for Subscription Watch access

### PR DESCRIPTION
**Pull request by kahowell**
For now our requirements are simply to grant access or not.

### CODE REVIEW

**Comment by kahowell** [comment]
Per UX request, I'm adjusting some capitalization... Will simply force-push an update in a moment.

**Comment by kahowell** [comment]
Okay... now I have the capitalization as requested :smile:

**Comment by kahowell** [comment]
I just got word that we might want to do `platform_default: true`, once I confirm, I'll come back to this PR and update if necessary. Please hold off on merge until then.

**Comment by kahowell** [comment]
I just force-pushed to tweak to platform_default to true, per the above comment. Also adjusted the commit message and PR title to reflect capitalization changes.

**Comment by kahowell** [comment]
@coderbydesign FWIW, this can be merged whenever is convenient for you.

File: configs/subscriptions.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "roles": [
+    {
+      "name": "Subscription Watch administrator",
+      "description": "Perform any available operation against any Subscription Watch resource.",
+      "system": true,
+      "platform_default": true,
+      "version": 1,
```
Comment (coderbydesign) [inline configs/subscriptions.json:8]: Hey @kahowell - due to an issue with how the seeding gets run, the initial "version" needs to actually be set to `2`.
```suggestion
      "version": 2,
```

**Pull request merged**